src/OSPSuite.Infrastructure/Serialization/ORM/MetaData/ObservedDataMetaData.cs

```diff
@@ -2,13 +2,16 @@
 
 namespace OSPSuite.Infrastructure.Serialization.ORM.MetaData
 {
-   public class ObservedDataMetaData: MetaData<string>, IUpdatableFrom<ObservedDataMetaData>
+   public class ObservedDataMetaData : MetaData<string>, IUpdatableFrom<ObservedDataMetaData>
    {
-       public virtual DataRepositoryMetaData DataRepository { get; set; }
+      public virtual DataRepositoryMetaData DataRepository { get; set; }
 
       public void UpdateFrom(ObservedDataMetaData source, ISession session)
       {
-         DataRepository = source.DataRepository;
+         if (DataRepository == null)
```

Comment:
The problem was that the reference was always updated, thus creating a new MetaDataContent object every time!!
(source being the object created on save that should be used to update the existing objects instead of creating new ones).